content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -180,43 +332,51 @@ To create a new Secret, perform the following steps:
      - command:
        - kube-apiserver
        ...
-       - --encryption-provider-config=/etc/kubernetes/enc/enc.yaml  # <-- add this line
+       - --encryption-provider-config=/etc/kubernetes/enc/enc.yaml  # add this line
        volumeMounts:
        ...
-       - name: enc                           # <-- add this line
-         mountPath: /etc/kubernetes/enc      # <-- add this line
-         readonly: true                      # <-- add this line
+       - name: enc                           # add this line
+         mountPath: /etc/kubernetes/enc      # add this line
+         readonly: true                      # add this line
        ...
      volumes:
      ...
-     - name: enc                             # <-- add this line
-       hostPath:                             # <-- add this line
-         path: /etc/kubernetes/enc           # <-- add this line
-         type: DirectoryOrCreate             # <-- add this line
+     - name: enc                             # add this line
+       hostPath:                             # add this line
+         path: /etc/kubernetes/enc           # add this line
+         type: DirectoryOrCreate             # add this line
      ...
    ```
 
-1. Restart your API server.
+1. Restart the API server; for example, using `crictl stopp`.
```

Comment:
No experience on `crictl`, is `crictl stopp` a typo? Please confirm.

Reply:
Actually, we don't need to do this. The static pod will be recreated if the YAML file is changed. Right?

Reply:
`stopp` is not a typo.

We don't know how often the kubelet polls for static pod manifests. The default sync frequency, 60s, is not too bad, but it might be quite a bit higher.